sector-builder-ffi/examples/simple.rs

```diff
@@ -255,12 +258,25 @@ unsafe fn sector_builder_lifecycle(use_live_store: bool) -> Result<(), Box<Error
     // drop the first sector builder, relinquishing any locks on persistence
     sector_builder_ffi_destroy_sector_builder(sector_builder_a);
 
-    // create a new sector builder using same prover id, which should
-    // initialize with metadata persisted by previous sector builder
+    // migrate staged sectors, sealed sectors, and sector builder metadata to
+    // new directory (overwrites destination directory)
+    let renames = vec![
+        (metadata_dir_a.as_ref(), metadata_dir_b.as_ref()),
+        (staging_dir_a.as_ref(), staging_dir_b.as_ref()),
+        (sealed_dir_a.as_ref(), sealed_dir_b.as_ref()),
+    ];
+
+    for (from, to) in renames {
+        fs::rename(from, to).expect(&format!("could not rename from {:?} to {:?}", from, to));
```

Comment:
renames can easily fail, so shouldn't panic, but bubble up the error

Reply:
You are correct - but this is test code. If it fails, I _want_ it to panic and fail the test! :)